src/Lykke.SettingsReader/SettingsProcessor/Checkers/SqlChecker.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Reflection;
+using System.Data.SqlClient;
+using Lykke.SettingsReader.Exceptions;
+
+namespace Lykke.SettingsReader.Checkers
+{
+    internal class SqlChecker : ISettingsFieldChecker
+    {
+        public CheckFieldResult CheckField(object model, PropertyInfo property, object value)
+        {
+            if (value == null)
+                throw new CheckFieldException(property.Name, value, "Setting can not be null");
+
+            string val = value.ToString();
+            if (string.IsNullOrWhiteSpace(val))
+                throw new CheckFieldException(property.Name, value, "Empty setting value");
+
+            string url = string.Empty;
+            try
+            {
+                var connection = new SqlConnection(val);
```

Comment:
i suppose its better to wrap it into using(var connection = new SqlConnection(val)) statement